app/src/main/java/com/ubclaunchpad/room8/SignUpActivity.java

```diff
@@ -83,6 +84,57 @@ private void registerUser() {
             mPassword.requestFocus();
             return;
         }
+        // check the password meets correct requirements
+        boolean capitalFlag = false;
```

Comment:
Better to pull out to a helper method, something like "validatePasswordRequirements()"